message_definitions/v1.0/minimal.xml

```diff
@@ -593,6 +620,12 @@
       <entry value="175" name="MAV_COMP_ID_GIMBAL6">
         <description>Gimbal #6.</description>
       </entry>
+      <entry value="180" name="MAV_COMP_ID_BATTERY">
+        <description>Battery #1.</description>
+      </entry>
+      <entry value="181" name="MAV_COMP_ID_BATTERY2">
```

Comment:
I really don't get why we're bothering to name components.  There are just too great a variety/count of devices out there....

e.g. ArduPilot supports 9 batteries, each of which could be mavlink, and each of which ArduPilot could theoretically handle messages for...